src/me/mrCookieSlime/HotbarPets/HotbarPets.java

```diff
@@ -201,7 +197,7 @@ public boolean onRightClick(ItemUseEvent e, Player p, ItemStack item) {
                 }
             });
 
-            final HotbarPet shulker = new HotbarPet(new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYjFkMzUzNGQyMWZlODQ5OTI2MmRlODdhZmZiZWFjNGQyNWZmZGUzNWM4YmRjYTA2OWU2MWUxNzg3ZmYyZiJ9fX0=="), "&dShulker Pet", new String[]{"&cMob (Hostile)", "&7Favourite Food: Chorus Fruit", "", "&rRight-Click: &7Gives Levitation and Slow falling"}), "HOTBAR_PET_SHULKER", new ItemStack(Material.CHORUS_FRUIT), new ItemStack[] {
+            final HotbarPet shulker = new HotbarPet(new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYjFkMzUzNGQyMWZlODQ5OTI2MmRlODdhZmZiZWFjNGQyNWZmZGUzNWM4YmRjYTA2OWU2MWUxNzg3ZmYyZiJ9fX0=="), "&dShulker Pet", new String[]{"&cMob (Hostile)", "&7Favourite Food: Chorus Fruit", "", "&rRight-Click: &7Shoots tipped arrows"}), "HOTBAR_PET_SHULKER", new ItemStack(Material.CHORUS_FRUIT), new ItemStack[] {
```

Comment:
You should maybe also add what kind of tipped arrow is shot.

Reply:
Done